src/templates/pycontw-2020/contents/en/event/open-spaces.html

```diff
@@ -0,0 +1,59 @@
+{% extends 'contents/_base.html' %}
+
+{% block title %}Open Spaces{% endblock %}
+
+{% block content %}
+
+<h1>Open Spaces</h1>
+
+<p>Open Spaces are self-organizing meetup-like events which happen in parallel with the main conference. They are held in Multifunction Room(1F) of the conference venue.  While most of the conference is planned months in advance, Open Spaces are created “just in time” by PyCon attendees during the conference. They provide a way for you to self-gather, self-define, and self-organize any way you like.</p>
```

Comment:
The word `Multifunction Room` is not used consistenly in this document. It would be better if we can unify them.

Reply:
Is there already a word for the open-spaces location?  Multifunction Room is referenced from NCKU en website.

Reply:
The wording **Multifunction Room** is good. What I mean is that we should use the exact word on the page. There are **multifunction room**, **Multifunction** and etc. I think we can just use **Multifunction Room**.

Reply:
OK, wording fixed~